src/core.js

```diff
@@ -56,11 +58,14 @@ extend( QUnit, {
 			var module = {
 				name: moduleName,
 				parentModule: parentModule,
-				tests: []
+				tests: [],
+				testsRun: 0
```

Comment:
I'll open a PR documenting the new `before`/`after` hooks for `module`. Is there anything else I should document?